criu/kerndat.c

```diff
@@ -863,7 +863,8 @@ static void kerndat_save_cache(void)
 {
 	int fd, ret;
 	struct statfs s;
-
+	ret= access(KERNDAT_CACHE_FILE_TMP, R_OK | W_OK | X_OK | F_OK );
+	pr_msg("open acess: %d, %s  \n",ret,strerror(errno));
```

Comment:
Please, drop this change. Missing permissions are found out by EACCESS errno of open.